docs/reference/release-notes/7.16.asciidoc

```diff
@@ -130,6 +136,36 @@ Monitoring::
 * Deprecate `create_legacy_templates` setting on HTTP exporters {es-pull}78564[#78564]
 * Deprecate the monitoring ingest settings {es-pull}77659[#77659] (issue: {es-issue}77459[#77459])
 
+[[known-issues-7.16.0]]
+[discrete]
+=== Known issues
+
+// tag::painless-functional-interface-resolution-81696-known-issue[]
+* Painless function interface resolution: Any Painless script
+where a method call resolves a lambda argument to a `UnaryOperator`
+or a `BinaryOperator` results in an `IllegalArgumentException` as either
+a compilation error when static types are used or a runtime error when
+dynamic types are used. Use standard procedural statements as a workaround
+ (`if/else`, `while`, `for`, etc.).
++
+An example failure of a `UnaryOperator`:
++
+[source,Java]
+----
+List l = ['abc', '123']; l.replaceAll(f -> f.replace('abc', 'doremi');
+----
++
+The workaround:
```

Comment:
```suggestion
The following example is a possible workaround:
```

Reply:
I thought about introducing "a possible workaround" instead of "the workaround" because we indicate that different procedural statements can be used, instead of suggesting that this is a singular workaround.